src/Base/AbstractResource.php

```diff
@@ -130,7 +138,7 @@ public function store(Request $request)
 
         if ($request->ajax()) {
             return [
```

Comment:
'return response()->json()' could be used here.

I know, that Laravel will automatically transform the response based on the passed headers, so it really doesn't matter. It's just my personal preference, so you can ignore this if you want to